.eslintrc.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     "array-callback-return": "off",
     "linebreak-style": "off",
     "no-console": 1,
-    "import/no-extraneous-dependencies": ["error", { "devDependencies": ['**/*test.js', '**/testcafeRunner.js'] }]
+    "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/test-utils/**", '**/*test.js', '**/testcafeRunner.js'] }]
```

Comment:
do we still need this?

Reply:
Thought as I added it in to match MP i might as well keep them consistent